analyses/doublet-detection/scripts/01a_run-scdblfinder.R

```diff
@@ -67,18 +67,53 @@ run_scdblfinder <- function(sce,
 }
 
 
+#' Export a TSV of all NA values (except barcodes), with expected scDblFinder output column names
+#'
+#' @param barcodes Values for the `barcodes` column
+#' @param output_tsv_file Output TSV file path
+export_na_tsv <- function(barcodes, output_tsv_file) {
+    output_colnames <- c(
+      "barcodes",
+      "type",
+      "weighted",
+      "distanceToNearest",
+      "distanceToNearestDoublet",
+      "distanceToNearestReal",
+      "nearestClass",
+      "ratio.k3",
+      "ratio.k10",
+      "ratio.k15",
+      "ratio.k20",
+      "ratio.k25",
+      "lsizes",
+      "nfeatures",
+      "nAbove2",
+      "src",
+      "cxds_score",
+      "include.in.training",
+      "score",
+      "class"
```

Comment:
Ah there are a lot of columns here! Do we want to output everything? I think we could probably just output `score` and `class`? Those appear to be the most relevant (see https://github.com/plger/scDblFinder?tab=readme-ov-file#basic-usage) and I don't think we need these remaining stats...

Reply:
We will need to update the function for running `scDblFinder` to only return those columns.